.github/workflows/linters.yml

```diff
@@ -1,5 +1,5 @@
 name: Linters
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Wouldn't that mean linters won't run if you push commit extra commit to your PR ? I think we still want that functionality. Maybe we need to move to a different reviewdog runner : https://github.com/reviewdog/reviewdog?tab=readme-ov-file#reporter-github-checks--reportergithub-check
@mkllnk  @dacook what do you think ?